pkg/csi_driver/multishare_ops_manager.go

```diff
@@ -648,7 +649,7 @@ func containsOpWithShareName(shareName string, opType util.OperationType, ops []
 func containsOpWithShareTarget(share *file.Share, opType util.OperationType, ops []*OpInfo) (*OpInfo, error) {
 	shareUri, err := file.GenerateShareURI(share)
 	if err != nil {
-		return nil, status.Errorf(codes.Internal, "failed to parse share handle, err: %v", err.Error())
```

Comment:
@saikat-royc Should all of the errors returned from GenerateShareURI really be an internal error? Or can "missing share parent instance" or "missing parent, project, name or location in share parent" be InvalidArgument?